WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -210,6 +206,9 @@ import WordPressComAnalytics
         // pull to refresh animation.
         view.userInteractionEnabled = readerTopic != nil
 
+        NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(defaultAccountDidChange(_:)), name: WPAccountDefaultWordPressComAccountChangedNotification, object: nil)
```

Comment:
I'm wondering if we can remove the observer completely.  It was useful when the post list was the root vc of the reader as it was always recycled, but with the recent menu refresh that's no longer the case.  All post streams are discarded during login/logout.   I suppose it might depend on how we're handing the split view integration.